modules/internal/Bytes.chpl

```diff
@@ -677,7 +677,7 @@ module Bytes {
     }
 
     /*
-      Splits the :record:`bytes` on `sep` yielding the subbytes between each
+      Splits the :record:`bytes` on `sep` yielding the bytes between each
```

Comment:
I'm just guessing this is a search-and-replace of `substring`, and that there isn't a concept of a `subbytes`.